src/backend/base/langflow/components/inputs/chat.py

```diff
@@ -13,7 +13,7 @@ class ChatInput(ChatComponent):
     name = "ChatInput"
 
     inputs = [
-        MultilineInput(
+        StrInput(
```

Comment:
@raphaelchristi Could we do this instead:

1. Keep it as a MultilineInput as before
2. But, pass `input_types=[]` so that the handle should disappear

Multiline has some nice properties when composing the text, so i think if we do this, we get the advantage of your PR (the handle being hidden for UI cleanliness) but also the multiline advantage.

Reply:
Thank you @erichare for this suggestion! I agree this is a much better approach.